lib/sqlalchemy/engine/result.py

```diff
@@ -2010,7 +2043,7 @@ class MappingResult(_WithKeys, FilterResult[RowMapping]):
 
     _post_creational_filter = operator.attrgetter("_mapping")
 
-    def __init__(self, result: Result[Any]):
+    def __init__(self, result: Result[Unpack[_RawRowType]]):
```

Comment:
**Michael Bayer** (zzzeek) wrote:

_RawRowType should be TupleAny

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4986